### ISSUE TITLE
Saturated alerts have extra space

### ISSUE THREAD

**Issue by twant** [phetsims/molarity#195]
There is a space in "Less solute, less solids. Saturated ." at the end after "Saturated" and before the period.

**Comment by twant** [comment]
turns out we were missing qualitative state info. Pushing changes now and closing.
